addons/udes_stock/models/stock_move.py

```diff
@@ -1,10 +1,31 @@
 # -*- coding: utf-8 -*-
-from odoo import models, api, _
+from odoo import models, fields, api, _
+from odoo.addons import decimal_precision as dp
 
 
 class StockMove(models.Model):
     _inherit = "stock.move"
 
+    u_initial_demand = fields.Float("Initial Demand", digits="Product Unit of Measure")
+
+    product_uom_qty = fields.Float(string="Quantity")
```

Comment:
Why do we override?

Reply:
We're changing the string from "Demand" to "Quantity".

This is because we've added `u_initial_demand` which will just be set to the initial quantity. Then we always have a track of what this was when `product_uom_quantity` is updated, e.g. because of full package reservation.

Reply:
@caleb-shelton might be worth adding a comment above to make this clear